python/packages/aws-lambda-sdk/serverless_lambda_sdk/internal_extension/base.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import annotations
```

Comment:
@alexdelorenzo can you describe how Lambdas are expected to be set up to have that wrapping working?

After we package the Python SDK into a layer and attach it to the function, what environment variables with what values need to be ensured to have auto-wrapping work?

Reply:
Ah, didn't know that was in scope for this, but I've added functionality that addresses this. Users need to set `AWS_LAMBDA_EXEC_WRAPPER` to `/opt/serverless_aws_lambda_sdk/internal_extension/exec_wrapper`, assuming that path is how this SDK will be packaged for the Lambda extension layer.

Reply:
> Ah, didn't know that was in scope for this

Without that, wrapper cannot really be utilized, so yes it needs to be complete in a sense that after merging it we should be able to wrap any Python function with it (otherwise how can we test that it works?)

> `AWS_LAMBDA_EXEC_WRAPPER` to `/opt/serverless_aws_lambda_sdk/internal_extension/exec_wrapper`, 

In Node.js it's `/opt/sls-sdk-node/exec-wrapper.sh`. It'll be nice if for Python we follow sam naming convention. So put all extension logic into `/opt/sls-sdk-python`  and have internal extension script accessible at `/opt/sls-sdk-python/exec-wrapper.sh`.